src/orderbook/__tests__/state.spec.ts

```diff
@@ -0,0 +1,124 @@
+import { EventData } from 'web3-eth-contract'
+import { DEFAULT_ORDERBOOK_OPTIONS } from '..'
+import { AccountState } from '../state'
+
+function accountState(): AccountState {
+  return new AccountState(
+    { forEvent: (ev) => Promise.resolve(ev.blockNumber) },
+    { ...DEFAULT_ORDERBOOK_OPTIONS, strict: true },
+  )
+}
+
+function ev(batch: number, name: string, data: any, index: number = 0): EventData {
```

Comment:
whats the tiny word here? My only guess is event. Is there a reason we don't use the full word?

Reply:
`ev` is indeed for `event`. When I used `event` it had a keyword highlighting in my editor so I though it might have been a special reserved keyword in some flavour of JavaScript.

I also thought because the return type was `EventData` that the shorter name was OK.